src/useCapabilitiesSafe.ts

```diff
@@ -0,0 +1,29 @@
+import { useAccount } from 'wagmi';
+import { useCapabilities } from 'wagmi/experimental';
+import type { WalletCapabilities } from './types';
+
+export function useCapabilitiesSafe({
+  chainId,
+}: {
+  chainId: number;
+}): WalletCapabilities {
+  const { isConnected } = useAccount();
+
+  const { data: capabilities, error } = useCapabilities({
+    query: { enabled: isConnected },
+  });
```

Comment:
I believe you mentioned this if fails logs an error.

I think the all point of this wrapper, is to improve the developer experience, and have those error being gone. Otherwise folks will ping us over Twitter and say, "I see an error, did I do something wrong?", when the reality is an exptected error.

So ideally we want to get into the habit that errors that are predicted, and the user can't do nothing about it, we hide them with a try/catch.

@0xAlec what do you think?